build/package.proj

```diff
@@ -23,7 +26,7 @@
 
   <Target Name="_Package" DependsOnTargets="$(_PackageDependsOn)" />
 
-  <Target Name="_CreateNuGetPackages">
+  <Target Name="_CreateNuGetPackages" DependsOnTargets="_GenerateBundledVersionsProps">
```

Comment:
Why not move this to the above PropertyGroup so that all the dependencies are in one place and easy to find?

```
     <_PackageDependsOn>
       $(_PackageDependsOn);
         _GenerateBundledVersionsProp;
       _CreateNuGetPackages;
     </_PackageDependsOn>
```

Reply:
Logically _CreateNuGetPackages requires _GenerateBundledVersionsProps to run otherwise it would be missing files, hence the DependsOnTargets. I'll add _GenerateBundledVersionsProps to _PackageDepends on so it's clear that it's also run during pack.